src/fragments/map-view/components/ors-l-polyline/ors-l-polyline.js

```diff
@@ -203,11 +203,9 @@ export default {
         }
         if (tooltipIcon) {
           let tooltip = `
-          <div>
-            <div style='min-width:30px;width:20%;height:50px;float:left'>
-              <span style='${iconStyle}' class="material-icons">${tooltipIcon}</span>
-            </div>
-            <div style='min-width:180px'>${tooltipInnerContent}</div>
+          <div style='vertical-align: middle'>
```

Comment:
`vertical-align: middle` is only used on inline elements, not block elements.
I assume the idea here was to align contents of this div vertically - this should be on the `span`s then.